unfoldingWord/translationCore
# Root Folder Restructure #1774

#### This pull request addresses:

Root Folder Restructuring. Minimizes paths in the root.


#### How to test this pull request:

Launch the app and look for all images and splash loading screen should look normal.
- [ ] Default User Image
- [ ] Door43 icon
- [ ] tC icons on splash, welcome, alerts.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/1791)
